docs/learn/learn-account-abstraction.md

```diff
@@ -0,0 +1,61 @@
+---
+id: learn-account-abstraction
+title: Polkadot's Account Abstraction
+sidebar_label: Account Abstraction
+description: Polkadot's Native Account Abstraction.
+keywords: [account, polkadot account, account abstraction, proxy, multisig, batch]
+slug: ../learn-account-abstraction
+---
+
+## Your Keys, Your Responsibility
+
+Account abstraction addresses the challenges of managing cryptographic keys representing accounts on
+blockchains and, more generally, your digital identity. In Web3, you digitally
+sign any transaction or, more generally, any message using your private key. Data is recorded on a
+public ledger (blockchain) whose multiple copies of it are stored in computers participating in a
+P2P network.
+
+While the account keys grant users control and ownership, losing them results in losing access to digital assets and fragmentation of your digital identity since you will need to create a new account with a
+new set of keys. This poses a hurdle for both users and developers regarding security and
```

Comment:
It could be we emphasize that the user is _still_ responsible for their keys - that has not changed, but that they can take precautions using the functionality that Polkadot provides in order to ensure they don't end up losing their accounts.